File: message-bus/src/main/java/alma/obops/draws/messages/rabbitmq/RabbitMqMessageQueue.java
package alma.obops.draws.messages.rabbitmq;

import alma.obops.draws.messages.MessageBroker;
import alma.obops.draws.messages.MessageQueue;

/**
 * @author mchavan  11-Oct-2018
 */
public class RabbitMqMessageQueue extends MessageQueue {

	private String rmqQueueName;

	public RabbitMqMessageQueue( String queueName, MessageBroker broker, String rmqQueueName ) {
		super( queueName, broker );
		this.rmqQueueName = rmqQueueName;
	}

	public String getRmqQueueName() {
		return rmqQueueName;
	}
}

File: message-bus/src/draws/messages/rabbitmq/RabbitMqMessageQueue.py
from draws.messages.MessageQueue import MessageQueue

class RabbitMqMessageQueue(MessageQueue):
    def __init__(queueName, broker, rmqQueueName):
        super(queueName, broker)
        self.__rmqQueueName = rmqQueueName

    def getRmqQueueName(self):
        return self.__rmqQueueName;
